geniza/corpus/templates/admin/corpus/document/merge.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'admin/base_site.html' %}
```

Comment:
I wonder if there is a template closer to what we need? maybe the change form? (but possibly not, could end up overwriting a lot more)

Reply:
This template provides about the same amount of what's needed as the change_form.